ghost_adapter.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features|doc)/}) }
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features|doc|bin)/}) }
   end
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activerecord', '>= 5'
+  spec.add_dependency 'activerecord', '>= 5', '< 6.2'
```

Comment:
starting to test on 7.0 alpha already, but I didn't want to include it in here yet